annif/project.py

```diff
@@ -90,6 +102,7 @@ def initialize(self):
         logger.debug("Initializing project '%s'", self.project_id)
 
         self._initialize_analyzer()
+        self._initialize_transformer()  # TODO: Is this needed?
```

Comment:
I'm trying to decide whether to recommend keeping the initialize function or removing it. It would perhaps be helpful for catching configuration errors early. OTOH they will be encountered anyway in most operations (though probably not in commands like `list-projects`).

Perhaps it would be wise to apply the YAGNI principle. The current transformers (limit and pass) don't need any special initialization, so this can be dropped for now. It can always be added back later if some other transformer (for example machine translation) needs initializing.